src/java.base/share/classes/sun/security/provider/DigestBase.java

```diff
@@ -34,6 +40,10 @@
 import jdk.internal.util.Preconditions;
 import jdk.internal.vm.annotation.IntrinsicCandidate;
 
+/*[IF CRIU_SUPPORT]*/
+import openj9.internal.criu.CRIUSECProvider;
```

Comment:
An import for `InternalCRIUSupport` is needed.

Reply:
Updated

Reply:
Updated